frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -52,8 +55,19 @@ const ExpressionWidget = (props: ExpressionWidgetProps): JSX.Element => {
 
   return (
     <ExpressionWidgetWrapper>
-      <div className="p2">
-        <div className="h5 text-uppercase text-light text-bold">{t`Expression`}</div>
+      <div className="p3 pb4">
```

Comment:
For new components, we try to remove the usage of class utilities and favor using `emotion` instead. Could you extract any components that have the className changes to `ExpressionWidget.style.ts` instead?